exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -93,10 +94,13 @@ func newPRWExporter(cfg *Config, set exporter.CreateSettings) (*prwExporter, err
 
 // Start creates the prometheus client
 func (prwe *prwExporter) Start(ctx context.Context, host component.Host) (err error) {
+	prwe.settings.Logger.Info("Starting Prometheus Remote Write Exporter...")
 	prwe.client, err = prwe.clientSettings.ToClient(host, prwe.settings)
 	if err != nil {
+		prwe.settings.Logger.Error("Failed to create HTTP client", zap.Error(err))
```

Comment:
we don't log errors we return as a rule of thumb.

Reply:
What would be a good way to let users know something went wrong for example a typo in the hostname which would result in a 404. Or if you forgot to enable the push write functionality? The error gives clear information about that

Reply:
Does this error get logged by the caller anyways? Are we just duplicating logging?

Reply:
What do you mean with duplicating logging?

Reply:
When this error is returned, does the caller log it?

Reply:
Caller logs it, added it there as well. So this logging can be removed then.